commit bb2caafa82f1210ca83aae1753b9c992efee6871 (refs/changes/43/2043/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-06-28T23:21:44+00:00 (2 years, 3 months ago)
    
    MANTA-3300 marlin crash due to cueball#111
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/dev/package.json b/dev/package.json
index ff2b4cb..90a7df6 100644
--- a/dev/package.json
+++ b/dev/package.json
@@ -16,7 +16,7 @@
 		"assert-plus": "^1.0.0",
 		"bunyan": "0.20.0",
 		"catstreams": "0.5.0",
-		"cueball": "2.2.6",
+		"cueball": "2.2.7",
 		"dtrace-provider": "0.6.0",
 		"extsprintf": "^1.3.0",
 		"fuzzstream": "0.0.1",
